core/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -90,53 +54,30 @@ oppia.factory('CollectionLinearizerService', [
       return index;
     };
 
-    var getNodeLeftOf = function(linearNodeList, nodeIndex) {
-      return nodeIndex > 0 ? linearNodeList[nodeIndex - 1] : null;
+    var getNodeIndexLeftOf = function(linearNodeList, nodeIndex) {
+      return nodeIndex > 0 ? nodeIndex - 1 : null;
     };
 
-    var getNodeRightOf = function(linearNodeList, nodeIndex) {
+    var getNodeIndexRightOf = function(linearNodeList, nodeIndex) {
       var lastIndex = linearNodeList.length - 1;
-      return nodeIndex < lastIndex ? linearNodeList[nodeIndex + 1] : null;
+      return nodeIndex < lastIndex ? nodeIndex + 1 : null;
     };
 
     // Swap the node at the specified index with the node immediately to the
     // left of it.
     var swapLeft = function(collection, linearNodeList, nodeIndex) {
-      // There are three cases when swapping a node left:
-      //   1. There is a node right of it but not left
-      //   2. There is a node left of it but not right
-      //   3. There are nodes both to the left and right of it
+      // There is a single case in the swap
```

Comment:
I don't think you need this comment (or the line below).

Reply:
Done, removed those.